src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions/StringExtensions.cs

```diff
@@ -34,6 +34,20 @@ internal static partial class StringExtensions
 		private static readonly Lazy<Regex> _newLineRegex = new Lazy<Regex>(() => new Regex(@"^", RegexOptions.Multiline));
 #endif
 
+		public static bool TryGetStringByRange(this string instance, int start, int len, out string result)
+		{
+			result = null;
+			
+			if (!string.IsNullOrEmpty(instance) && instance.Skip(start).Take(len).ToArray().Length == 0)
```

Comment:
If `instance` is null, this will throw an `ArgumentNullException`.

`!string.IsNullOrEmpty(instance)` will evaluate to false, then we'll try to do `instance.Skip(start)` and `Skip` extension will throw for the null `instance`.